index.js

```diff
@@ -0,0 +1,151 @@
+// task__1
+
+const sayHello = function (name) {
+    if (name == 'Mark') {
+        return console.log(`Hi, ${name}`);
+    } else {
+        return console.log(`Hello, ${name}`);
+    }
+};
+
+// const sayHello = name => {
+//     if (name == 'Mark') {
+//         return console.log(`Hi, ${name}`);
+//     } else {
+//         return console.log(`Hello, ${name}`);
+//     }
+// };
+
+sayHello('Mark');
+sayHello('Oleg');
+sayHello('Viktor');
+
+
+
+
+// task__2 
+
+const result = function(n, m) {
+    return Math.sqrt((n * n) + (m * m));
+};
+
+// const result = (n, m) => {
+//     return Math.sqrt((n * n) + (m * m));
+// }
+
+console.log(result(3, 4));
+
+
+
+
+
+// task__3
+
+let min = function(a, b) {
+    if (a < b) {
+        return a;
+    } else {
+        return b;
+    }
+};
```

Comment:
If you found out the global object `Math` in previous task, you will be very happy that there is `Math.min(a. b)` method, whish solve your problem